Shared/CoreData/Chain+Shared.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Chain+SharedFetchRequest.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 26.07.21.
+//  Copyright © 2021 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+extension NSFetchRequest where ResultType == Chain {
```

Comment:
Could we add a comment here that this file is shared between different targets?

Reply:
sure